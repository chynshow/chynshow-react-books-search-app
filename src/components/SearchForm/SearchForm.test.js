import React from 'react';
import SearchFrom from './';
import { shallow } from 'enzyme';

it('should renders SearchFrom component', () => {
  const c = shallow(<SearchFrom />);
  expect(c).toMatchSnapshot();
});
