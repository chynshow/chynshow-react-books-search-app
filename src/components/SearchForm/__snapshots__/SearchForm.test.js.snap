// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders SearchFrom component 1`] = `
<form
  className="c-search-form"
  onSubmit={[Function]}
>
  <Memo(Input)
    className="c-search-form__input"
    name="query"
    onChange={[Function]}
    placeholder="Your search query"
    value=""
  />
  <Memo(Input)
    className="c-search-form__input"
    name="inauthor"
    onChange={[Function]}
    placeholder="Author"
    value=""
  />
  <Memo(Input)
    className="c-search-form__input"
    name="inpublisher"
    onChange={[Function]}
    placeholder="Publisher"
    value=""
  />
  <Memo(Input)
    className="c-search-form__input"
    name="isbn"
    onChange={[Function]}
    placeholder="ISBN"
    type="number"
    value=""
  />
  <button
    className="c-btn c-search-form__btn"
  >
    Search
  </button>
</form>
`;
